.github/workflows/haskell.yml

```
@@ -243,7 +243,8 @@ jobs:
     - name: Run tests
       run: |
         export CARDANO_MAINNET_MIRROR="$(pwd)/epochs"
-        cabal test ${{ matrix.package }}
+        if [ -z "$NIGHTLY" ]; then TRIES=1; else TRIES=2; fi
+        scripts/cabal-test-with-retries.sh "${{ matrix.package }}" "$TRIES"
```

We don't need to force people reading that horrible ugly bash script if they don't care about nightly tests.
```suggestion
        if [ -z "$NIGHTLY" ]; then
          cabal test ${{ matrix.package }}
        else
          TRIES=2
          scripts/cabal-test-with-retries.sh "${{ matrix.package }}" "$TRIES"
        fi         
```

---

Yes, that seems better. I was going to do it that way originally but thought it would be more messy in the workflow file than it actually is, and the script ended up being more messy than I expected :-)

---

I also simplified the script now that it doesn't have to handle the `TRIES=1` case cleanly